diagnostic.md

```diff
@@ -40,31 +70,38 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    The serializer defines the attributes of the model, and allows the attributes to be saved on the database
```

Comment:
Serializers filter and reshape data.  They do not define the attributes of a model.